cmd/fs-v1-multipart.go

```diff
@@ -688,10 +693,14 @@ func (fs *FSObjects) AbortMultipartUpload(ctx context.Context, bucket, object, u
 		}
 		return toObjectErr(err, bucket, object)
 	}
+
 	// Ignore the error returned as Windows fails to remove directory if a file in it
 	// is Open()ed by the backgroundAppend()
 	fsRemoveAll(ctx, uploadIDDir)
 
+	// Remove the folder created in multipart
+	fsRemoveDir(ctx, fs.getUploadIDDir(bucket, object, ""))
```

Comment:
do
```
// It is safe to ignore any directory not empty error (in case there were multiple uploadIDs on the same object)
fsRemoveDir(ctx, fs.getMultipartSHADir(bucket, object))
```

Reply:
okay